Controllers/FHIRcastClientController.cs

```diff
@@ -87,21 +87,36 @@ public IActionResult Post([FromForm] ClientModel model)
         /// <param name="verification">Hub's verification response to our subscription attempt</param>
         /// <returns></returns>
         [HttpGet("{subscriptionId}")]
-        public IActionResult Get(string subscriptionId, [FromQuery] SubscriptionVerification verification)
+        public IActionResult Get(string subscriptionId)

```

Comment:
Did you see how validation is done when posting subscriptions? The validation is basically set on [the model][model] using attributes specifying what is required etc. The framework then generates error messages that [can be used in the response][response] (which should probably be `Bad request` and not `Not found`).

[model]: https://github.com/fhircast/sandbox/blob/4fa8a3f3a7fb7bc356a606154e38d6524fb72a9f/FHIRCastSandbox/Model/Subscriptions.cs#L9
[response]: https://github.com/fhircast/sandbox/blob/4fa8a3f3a7fb7bc356a606154e38d6524fb72a9f/FHIRCastSandbox/Controllers/HubController.cs#L49